parsers/rwcpopmelody_parser.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python
+"""
+Translates the RWC-POP SMF (Synchronized Midi File) annotations to a set of
+JAMS files, keeping only the melody track from each MIDI file.
+
+The original dataset is described online at the following URL:
+    https://staff.aist.go.jp/m.goto/RWC-MDB/rwc-mdb-p.html
+
+To parse the entire dataset, you just need to provide the path to the folder
+containing the SMF files, with filenames in the form RM-P[0-9]{3}*.MID from
+RM-P001*.MID to RM-P100*.MID
+
+Example:
+./rwcpopmelody_parser.py ~/AIST.RWC-MDB-P-2001.SMF_SYNC -o ~/RWCPOP_melody_jams/
+
+"""
+
+__author__ = "J. Salamon"
+__copyright__ = "Copyright 2016, Music and Audio Research Lab (MARL)"
+__license__ = "GPL"
+__version__ = "1.0"
+__email__ = "justin.salamon@nyu.edu"
+
+import argparse
+import logging
+import os
+import time
+import midi
+import pretty_midi
+import pandas as pd
+
+import jams
+
+
+def fill_file_metadata(jam, metadata, n_track):
+    """Fills the global metada into the JAMS jam."""
+    jam.file_metadata.artist = metadata['Artist (Vocal)'][n_track]
+    jam.file_metadata.title = metadata['Title'][n_track]
+
+    d_str = metadata['Length'][n_track]
+    jam.file_metadata.duration = (float(d_str.split(":")[0]) * 60 +
```

Comment:
this is pretty dirty.  Since we're already using pandas here, we can do something like this instead:
```python
In [4]: pd.Timedelta('0:2:34').total_seconds()
Out[4]: 154.0
```
(only problem is that you'll have to pad in hours, but i generally prefer that to reinventing second-parsing.)